internal/api/auth.go

```diff
@@ -129,3 +134,22 @@ func BuildJWT(user int) (string, error) {
 	// возвращаем строку токена
 	return tokenString, nil
 }
+
+func GetUserID(tokenString string) (int, error) {
```

Comment:
Если эти функции вызываются только в контексте пакета, стоит их сделать не экспортируемыми.
```suggestion
func getUserID(tokenString string) (int, error) {
```